flytekit/clis/sdk_in_container/serve.py

```diff
@@ -0,0 +1,46 @@
+from concurrent import futures
+
+import click
+import grpc
+from flyteidl.service.external_plugin_service_pb2_grpc import add_ExternalPluginServiceServicer_to_server
+
+from flytekit.extend.backend.external_plugin_service import BackendPluginServer
+
+_serve_help = """Start a grpc server for the external plugin service."""
+
+
+@click.command("serve", help=_serve_help)
+@click.option(
+    "--port",
+    default="80",
```

Comment:
admin also use 80, maybe set nodeport to 30090
```bash
flyteadmin                   ClusterIP   10.96.119.241   <none>        80/TCP,81/TCP,87/TCP,10254/TCP             2d
```

Reply:
this is the pod port, right? Having this as port 80 might be annoying in local tests.

Also, this will be exposed through a Service, which we're doing in https://github.com/flyteorg/flyte/pull/3454. Reading that PR though I'm don't get why we need to use NodePort services. Isn't the external plugin service accessed only by propeller?

Reply:
with nodeport, people are able to submit the job from local. only for testing.